doc/frequently-asked-questions.rst

```diff
@@ -449,6 +449,51 @@ following to get the ``scientific`` package build with
 
    nix-build -A haskell.packages.integer-simple.ghc802.scientific
 
+How to get an environment with GHC and LLVM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default on x86_64 LLVM is not included with GHC:
```

Comment:
```suggestion
By default LLVM is not included with GHC unless it is required for native code generation:
```

(Although this will only be true once we merge the new update)